cmd/root_args.go

```diff
@@ -70,6 +76,18 @@ var envMapBool = map[*bool]boundEnvVar[bool]{
 		Env:         helpers.Ptr("CREATE_MISSING_TARGET_BRANCHES"),
 		Default:     helpers.Ptr(true),
 	},
+	&feedbackCommitStatus: {
+		Name:        "feedback-commit-status",
+		Description: "Enable feedback commit status",
+		Env:         helpers.Ptr("FEEDBACK_COMMIT_STATUS"),
+		Default:     helpers.Ptr(true),
+	},
+	&fetchRateLimits: {
+		Name:        "fetch-rate-limits",
+		Description: "Enable per-event fetching of rate limits and corresponding logs decoration",
```

Comment:
Given that _every_ interaction with the GitHub API returns current usage in the response headers, I feel we're just introducing potential issues (i.e. hitting secondary rate limits) by directly polling the rate limit endpoint…